style.css

```diff
@@ -89,4 +89,284 @@ body {
+  }
+
+  /* About Section */
+
+  .container-about {
+    padding: 114px 24px;
+    align-items: center;
+    margin-top: 100px;
+    background: #fff;
+    border-top-right-radius: 120px;
+  }
+
+  .about-project-title {
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: left;
+    color: #172b4d;
+  }
+
+  .description-about {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #344563;
+    margin-top: 10px;
+  }
+
+  .skill-list,
+  .lang {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+    padding-top: 30px;
+  }
+
+  .lang li a {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 20px;
+    background-color: #f7f7f9;
+    border-radius: 5px;
+    text-decoration: none;
+    color: #253858;
+    padding: 5px;
+    font-size: 20px;
+    font-weight: 500;
+  }
+
+  .list {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .list span {
+    color: #000;
+    font-size: 20px;
+    font-weight: 500;
+  }
+
+  hr {
+    background: #dfe1e6;
+    height: 1px;
+    border: 0.5px;
+    margin-top: 10px;
+  }
+
+  .contact-me {
+    background: url(Assets/images/contact-form-shapes-mobile.svg) #6070ff;
+    background-repeat: no-repeat;
+    border-top-left-radius: 120px;
+    padding: 114px 24px;
+    background-position: 100% 10%;
+    height: max-content;
+  }
+
+  .contact-me h1 {
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .contact-me p {
+    text-align: center;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #fff;
+    margin-top: 10px;
+  }
+
+  .contact-form {
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    margin-top: 30px;
+  }
+
+  .btn.btn-default.btn-hover-2.btn-pressed {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    padding: 12px 16px;
+    width: 145px;
+    background-color: #fff;
+    border-radius: 8px;
+    border: 1px solid #6070ff;
+    color: #6070ff;
+    font-size: 17px;
+    margin-bottom: 80px;
+  }
+
+  #message {
+    height: 170px;
+  }
+
+  input[type=text],
+  input[type=email],
+  select,
+  textarea {
+    font-family: "Poppins", sans-serif;
+    width: 100%;
+    padding: 12px;
+    box-sizing: border-box;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    border-radius: 10px;
+    border: 1px solid #cfd8dc;
+  }
+
+  .slider-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .slider-image {
+    max-width: 100%;
+    max-height: 100%;
+  }
+
+  form {
+    margin-top: 2%;
+  }
+
+  /* BUTTON STYLES */
+  .submit-btn:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .submit-btn:active {
+    transform: scale(1.1);
+    background: #ffc898;
+    color: #6070ff;
+    -webkit-transform: scale(1);
+  }
```

Comment:
As requested by the previous reviewer, please let's follow the button interactions closely, such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template.

![image](https://github.com/lucy-sees/Portfolio/assets/78251927/19e79200-015a-41b8-84b6-61430c91221c)

**background: #2230D2;**